pkg/filter/wasmhost/hostfunc.go

```diff
@@ -380,6 +382,14 @@ func (vm *WasmVM) hostLog(level int32, addr int32) {
 	}
 }
 
+func (vm *WasmVM) hostNow() int64 {
+	return time.Now().UnixNano() / 1e6
```

Comment:
Why not just export nanosecond.

Reply:
there are two reasons:
1. The host functions need to support many languages, so we have to choose the lowest precision. 
2. Nanosecond requires 64 bits integers, but in some languages,  all numeric values are IEEE754 doubles and the max safe integer value is `2^53-1`, which is not larger enough to represent time in nanosecond (about 104 days).

Reply:
The put the time unit into the function name of document.

Reply:
updated